sdk/identity/Azure.Identity/src/FileSystemService.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+
+namespace Azure.Identity
+{
+    internal class FileSystemService : IFileSystemService
```

Comment:
If we're adding these we need to move CLI credentials to use them too and write more tests for it.

Reply:
Yes, that's the plan after this PR

Reply:
Actually, I think I'll add tests for `ProcessRunner` into this PR

Reply:
Done